Workade/Views&ViewModels/Magazine/ShareView.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  ShareView.swift
+//  Workade
+//
+//  Created by ryu hyunsun on 2022/11/22.
+//
+
+import UIKit
+import KakaoSDKTemplate
+import KakaoSDKCommon
+import KakaoSDKShare
+
+class ShareView: UIView {
+    var magazine: MagazineModel
+    weak var delegate: MagazineViewShareDelegate?
+    
+    private lazy var divider: UIView = {
+        let divider = UIView()
+        divider.backgroundColor = .rgb(0xF2F2F7)
+        divider.translatesAutoresizingMaskIntoConstraints = false
+        
+        return divider
+    }()
+    
+    private lazy var shareViewContrainer: UIView = {
+        let shareViewContrainer = UIView()
+        shareViewContrainer.translatesAutoresizingMaskIntoConstraints = false
+        
+        return shareViewContrainer
+    }()
+    
+    private lazy var shareLabel: UILabel = {
+        let label = UILabel()
+        label.text = "주변에 공유하기"
+        label.font = .customFont(for: .subHeadline)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        
+        return label
+    }()
+    
+    private lazy var buttonsContainer: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        
+        return view
+    }()
+    
+    private lazy var kakaoShareButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: "KakaoButton"), for: .normal)
```

Comment:
```suggestion
        button.setImage(UIImage(named: "KaKaoButton"), for: .normal)
```

해당 부분때문에 카카오 공유 이미지가 안뜬것 같습니당 수정해주세용

Reply:
수정되었습니다. Asset에 넣어놓고 파일 이름을 바꿨었는데 거기서 문제가 생겼던 것 같습니다. 감사합니다